framework/helpers/BaseJsHelper.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * @link http://www.yiiframework.com/
+ * @copyright Copyright (c) 2008 Yii Software LLC
+ * @license http://www.yiiframework.com/license/
+ */
+
+namespace yii\helpers;
+
+use yii\web\JsExpression;
+
+/**
+ * BaseJsHelper provides concrete implementation for [[JsHelper]].
+ *
+ * Do not use BaseJsHelper. Use [[JsHelper]] instead.
+ *
+ * @author Dima Naumenko <d.naumenko.a@gmail.com>
+ * @since 2.0.5
+ */
+class BaseJsHelper
+{
+    /**
+     * Escapes regular expression to use in JavaScript (client-side)
+     * @param string $regexp
+     * @return JsExpression
+     */
+    public static function escapeRegexp ($regexp) {
```

Comment:
Name should be full: `escapeRegularExpression` just like `RegularExpressionValidator`